src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -7551,14 +7531,7 @@ void SizeVRF(EnergyPlusData &state, int const VRFTUNum)
     std::string SizingString; // input field sizing description (e.g., Nominal Capacity)
     Real64 TempSize;          // autosized value of coil input field
     int FieldNum = 2;         // IDD numeric field number where input field description is found
-    int SizingMethod;       // Integer representation of sizing method name (e.g., CoolingAirflowSizing, HeatingAirflowSizing, CoolingCapacitySizing,
-                            // HeatingCapacitySizing, etc.)
-    bool PrintFlag = true;  // TRUE when sizing information is reported in the eio file
-    int zoneHVACIndex;      // index of zoneHVAC equipment sizing specification
-    int SAFMethod(0);       // supply air flow rate sizing method (SupplyAirFlowRate, FlowPerFloorArea, FractionOfAutosizedCoolingAirflow,
-                            // FractionOfAutosizedHeatingAirflow ...)
-    int CapSizingMethod(0); // capacity sizing methods (HeatingDesignCapacity, CapacityPerFloorArea, FractionOfAutosizedCoolingCapacity, and
-                            // FractionOfAutosizedHeatingCapacity )
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7558]:(style),[unreadVariable],Variable 'SAFMethod' is assigned a value that is never used.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7560]:(style),[unreadVariable],Variable 'CapSizingMethod' is assigned a value that is never used.

[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7558]:(style),[variableScope],The scope of the variable 'SAFMethod' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:7560]:(style),[variableScope],The scope of the variable 'CapSizingMethod' can be reduced.